app/presenters/licences/set-up.presenter.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/set-up` view licence set up page
+ * @module SetUpPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter.js')
+
+const roles = {
+  billing: 'billing',
+  workflowEditor: 'charge_version_workflow_editor',
+  workflowReviewer: 'charge_version_workflow_reviewer'
+}
+
+/**
+ * Formats data for the `/licences/{id}/set-up` view licence set up page
+ *
+ * @returns {Object} The data formatted for the view template
```

Comment:
Add in the missing JSDoc params

```suggestion
 * Formats data for the `/licences/{id}/set-up` view licence set up page
 *
 * @param {module:ChargeVersionModel[]} chargeVersions - All charge versions records for the licence
 * @param {module:WorkflowModel[]} workflows - All in-progress workflow records for the licence
 * @param {Object} auth - The auth object taken from `request.auth` containing user details
 * @param {Object} commonData - Licence data already formatted for the view's shared elements
 *
 * @returns {Object} The data formatted for the view template
```